wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -349,10 +346,48 @@ class Proto3WireProtocCompatibilityTests {
     assertJsonEquals(IDENTITY_ALL_TYPES_JSON, jsonPrinter.print(identityAllTypes))
   }
 
-  @Ignore("TODO")
   @Test fun serializeIdentityAllTypesMoshi() {
+    // Moshi prints empty lists and empty arrays.
+    val moshiJson = """{
```

Comment:
do we have a test to confirm that values like `0` inside of lists do get encoded?

Reply:
We have one for ProtoAdapter, not for the Json ones. I'll add one.

Reply:
added